src/main/java/com/ohdab/member/service/AddTeacherService.java

```diff
@@ -0,0 +1,52 @@
+package com.ohdab.member.service;
+
+import com.ohdab.member.exception.NoMemberException;
+import com.ohdab.member.repository.MemberRepository;
+import com.ohdab.member.service.dto.AddTeacherReqDto;
+import com.ohdab.member.service.dto.JoinReqDto;
+import com.ohdab.member.service.helper.MemberHelperService;
+import com.ohdab.member.service.usecase.AddTeacherUsecase;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@RequiredArgsConstructor
+@Transactional
+public class AddTeacherService implements AddTeacherUsecase {
+
+    private final MemberRepository memberRepository;
+    // private final JoinService joinService;
```

Comment:
```suggestion
    // private final JoinUsecase joinUsecase;
```
인터페이스로 빈 주입받고 테스트 돌려봐

Reply:
PasswordEncoder 부분에서 문제가 생김